css/style.css

```diff
@@ -0,0 +1,24 @@
+#premium {
+  display: flex;
+  flex-direction: row;
+}
+#header {
+  position: absolute;
+}
+#nav-bar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  top: 0px;
+
+  height: 20px;
+  justify-content: center;
+  align-items: center;
+  padding: 1px;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: lightblue;
```

Comment:
you should use the # tag for this element, so the background change could be replicated on the project

Reply:
_please take into consideration the 2 suggestions up_